docs/interacting_with_ansible.md

```diff
@@ -0,0 +1,90 @@
+# Interacting with Ansible
+
+When a dataplane service is executed during a role deployment, a corresponding
+[OpenStackAnsibleEE](https://openstack-k8s-operators.github.io/openstack-ansibleee-operator/openstack_ansibleee/)
+resource is created. The OpenStackAnsibleEE resource is the associated ansible
+execution with the service.
+
+OpenStackAnsibleEE resources are reconciled by
+[openstack-ansibleee-operator](https://github.com/openstack-k8s-operators/openstack-ansibleee-operator).
+During reconciliation a
+[Job](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/job-v1/)
+resource is created which in turn creates a
+[Pod](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/) resource. The pod is started with an [Ansible Execution Environment](https://docs.ansible.com/automation-controller/latest/html/userguide/execution_environments.html) image, and runs [ansible-runner](https://ansible.readthedocs.io/projects/runner/en/stable/).
+
+## Retrieving and inspecting OpenStackAnsibleEE resources
+
+During (or after) a deployment the instances of OpenStackAnsibleEE can be
+retrieved from the API.
+
+    oc get openstackansibleee
+
+Sample output when the default list of services:
+
+	NAME                                                  NETWORKATTACHMENTS   STATUS   MESSAGE
+	dataplane-deployment-configure-network-edpm-compute                        True     AnsibleExecutionJob complete
+	dataplane-deployment-configure-os-edpm-compute                             True     AnsibleExecutionJob complete
+	dataplane-deployment-install-os-edpm-compute                               True     AnsibleExecutionJob complete
+	dataplane-deployment-run-os-edpm-compute                                   True     AnsibleExecutionJob complete
+	dataplane-deployment-validate-network-edpm-compute                         True     AnsibleExecutionJob complete
+
+Querying for jobs, shows the corresponding job for each OpenStackAnsibleEE resource:
+
+	oc get jobs
+
+Sample output:
+
+	NAME                                                  COMPLETIONS   DURATION   AGE
+	dataplane-deployment-configure-network-edpm-compute   1/1           8s         2m51s
+	dataplane-deployment-configure-os-edpm-compute        1/1           8s         2m27s
+	dataplane-deployment-install-os-edpm-compute          1/1           8s         2m35s
+	dataplane-deployment-run-os-edpm-compute              1/1           8s         2m19s
+	dataplane-deployment-validate-network-edpm-compute    1/1           8s         2m43s
+
+Using the job name, the corresponding pod can be retrieved:
+
+	oc get pods | grep dataplane-deployment-configure-network-edpm-compute
+
+Sample output:
+
+	dataplane-deployment-configure-network-edpm-compute-2hshp   0/1     Completed            0                5m45s
+
+Using the pod name, the ansible logs can be retrieved:
+
+	oc logs dataplane-deployment-configure-network-edpm-compute-2hshp
```

Comment:
it is possible to get the logs from the job
`oc logs job.batch/dataplane-deployment-configure-network-edpm-compute`

Reply:
Thanks! I didn't know that :smile: